pkg/kubecli/ovm.go

```diff
@@ -88,6 +90,19 @@ func (o *ovm) Update(offlinevm *v1.OfflineVirtualMachine) (*v1.OfflineVirtualMac
 	return updatedOvm, err
 }
 
+func (v *ovm) Patch(name string, pt types.PatchType, data []byte, subresources ...string) (result *v1.OfflineVirtualMachine, err error) {
```

Comment:
Could you add a basic unit test with e.g. json-merge type to `pkg/kubecli/ovm_test.go`?

Reply:
+1

IIRC there were issues withc patching CRDs.